### PR TITLE
Updating lessons

### PR DESCRIPTION
**Pull request by enibigir**
Fixing:
Issue#22: Adding updated docker image "rootproject/root:6.22.06-conda"
Issue#16: Explaining YAML syntax of lists and dicts
Issue#13: Updating episode 02 Title

Upgrading actions/download-artifact@v2 to actions/download-artifact@v3

### CODE REVIEW

File: _episodes/08-adding-ci-to-your-code.md [hunk 1]
```diff
@@ -234,7 +234,7 @@ build_skim:
 
 # Building multiple versions
 
-Great, so we finally got it working... CI/CD isn't obviously powerful when you're only building one thing. Let's build both the version of the code we're testing and also test that the latest ROOT image (`rootproject/root-conda:latest`) works with our code. Call this new job `build_skim_latest`.
+Great, so we finally got it working... CI/CD isn't obviously powerful when you're only building one thing. Let's build both the version of the code we're testing and also test that the latest ROOT image (`rootproject/root:latest`) works with our code. Call this new job `build_skim_latest`.
```
Comment (klieret) [inline _episodes/08-adding-ci-to-your-code.md:237]: > CI/CD isn't obviously powerful when you're only building one thing.

Not sure how much sense this sentence makes to me, I think we can just delete it

**Review by klieret — APPROVED** [review]
Thanks a lot!

File: _episodes/08-adding-ci-to-your-code.md (hunk 1 above)
Comment (enibigir) [reply _episodes/08-adding-ci-to-your-code.md:237]: Agreed

**Comment by klieret** [comment]
I'll merge this now and fix the typo afterward

**Pull request merged**